internal/backend.go

```diff
@@ -228,21 +228,28 @@ func initializeObjects(pers persistence.Persistence) map[fans.Fan]controller.Fan
 	for config, fan := range initializeFans(controllers) {
 		updateRate := configuration.CurrentConfig.ControllerAdjustmentTickRate
 
-		var pidLoop util.PidLoop
-		if config.ControlLoop != nil {
```

Comment:
Technically we cannot remove this, but I am not very fond about keeping it forever if we change the structure now.
Maybe we can keep it and add a warning notification, so users are made aware that this is now deprecated.

Reply:
I have added a warning log for anyone still using this.